numpy/_pytesttester.py

```diff
@@ -169,7 +169,10 @@ def __call__(self, label='fast', verbose=1, extra_argv=None,
             pytest_args += ["--cov=" + module_path]
 
         if label == "fast":
-            pytest_args += ["-m", "not slow"]
+            # not importing at the top level to avoid circular import of module
+            from numpy.testing import IS_PYPY
+            pytest_args += ["-m", "not slow and not slow_pypy"] \
+                           if IS_PYPY else ["-m", "not slow"]
```

Comment:
I think this would be better as:
```
if IS_PYPY:
    pytest_args += ["-m", "not slow and not slow_pypy"]
else:
    pytest_args += ["-m", "not slow"]
```

Reply:
Or at least
```python
            pytest_args += [
                "-m", "not slow and not slow_pypy" if IS_PYPY else "not slow"]
```
but @mattip's version is clearer.